tools/lib/language.js

```diff
@@ -44,7 +44,12 @@ class Language {
   }
 
   get samplePath() {
-    return `./test/detect/${this.name}/default.txt`
+    if (!this.moduleDir) {
+      return `./test/detect/${this.name}/default.txt`;
+    }
+    else {
+      return `${this.moduleDir}/test/detect/${this.name}/default.txt`;
+    }
```

Comment:
```suggestion
    if (this.moduleDir) {
      return `${this.moduleDir}/test/detect/${this.name}/default.txt`;
    } else {
      return `./test/detect/${this.name}/default.txt`;    
    }
```

Avoid unnecessary negation.  Does this still look good?

Reply:
Looks fine - my usual practice is to place the common case first.
Want me to make that change?

Reply:
Yes, please.

IMHO: An if/else to me expressed equal weight on both branches... so I merely tried to simplify.  If I'd thought harder I might have turned the exception case into a guard and then left the "happy path" as the fall thru.

Reply:
Is it not enough to do the following for both cases?
```js
return `${this.moduleDir || "."}/test/detect/${this.name}/default.txt`; 
```
or slightly more verbose
```js
const prefix = this.moduleDir || ".";
return `${prefix}/test/detect/${this.name}/default.txt`; 
```

Reply:
Yeah, nice, but we already went one way. :)